# IKソルバーの設定ファイル
sekirei_arm:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.005 # 分解能(小さいほど精密だけど、計算コストが高くなる)
  kinematics_solver_timeout: 0.05
